config/config_helpers_test.go

```diff
@@ -560,120 +313,57 @@ func TestResolveEnvInterpolationConfigString(t *testing.T) {
 		include           *IncludeConfig
 		terragruntOptions *options.TerragruntOptions
 		expectedOut       string
-		expectedErr       error
+		expectedErr       string
```

Comment:
Just curious: Why did you switch to checking the string?

Reply:
Before, the error was returned directly from my code, so I could check it exactly. Now, the error is wrapped by the HCL2 parser, so doing an exactly equality check is harder. Therefore, I'm just doing a string contains check for the error name.